test/test_models.py

```diff
@@ -1,8 +1,10 @@
+import unittest
+
 import torch
 from torchaudio.models import Wav2Letter, _MelResNet
 
 
-class TestWav2Letter:
+class TestWav2Letter(unittest.TestCase):
```

Comment:
Instead of `unittest.TestCase`, can you use `TorchaudioTestCase` from `common_utils`?
It's available in the latest master.